src/Components/Blazor/Blazor/src/Rendering/RendererRegistry.cs

```diff
@@ -15,23 +16,39 @@ internal static class RendererRegistry
 
         private static int _nextId;
         private static Dictionary<int, WebAssemblyRenderer> _renderers = new Dictionary<int, WebAssemblyRenderer>();
+        private readonly static bool _isWebAssembly = RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY"));
```

Comment:
In the `blazor-wasm` branch the platform name is now `BROWSER` instead of `WEBASSEMBLY` so this will break when we merge.

However I agree your change is right as far as the `master` branch is concerned :) We'll just have to update it when we merge things.

I'm only raising this point to spread awareness about this change to the platform name.